config/qa-bulk-index.env

```diff
@@ -2,8 +2,12 @@ NYPL_API_BASE_URL=https://qa-platform.nypl.org/api/v0.1/
 NYPL_OAUTH_URL=https://isso.nypl.org/
 NYPL_OAUTH_KEY=AQECAHh7ea2tyZ6phZgT4B9BDKwguhlFtRC6hgt+7HbmeFsrsgAAAGswaQYJKoZIhvcNAQcGoFwwWgIBADBVBgkqhkiG9w0BBwEwHgYJYIZIAWUDBAEuMBEEDNQozUGkaz8WYD2lUAIBEIAo/SzNMA9LowO6gcnTUCcMjBaAU1RH/L3EAS14fjJCUpyZppkuEDUd7w==
 NYPL_OAUTH_SECRET=AQECAHh7ea2tyZ6phZgT4B9BDKwguhlFtRC6hgt+7HbmeFsrsgAAAIcwgYQGCSqGSIb3DQEHBqB3MHUCAQAwcAYJKoZIhvcNAQcBMB4GCWCGSAFlAwQBLjARBAxN3indXvk2ueiE6CwCARCAQ018FdIVXXwfTuKH1vp/ZTfjBinxKTDosMmzyWB9/CtiFtgOu09iiyZEpC3AyGOt8ExywHZoHOZQuLdGGNFgbusmldw=
-ELASTICSEARCH_CONNECTION_URI=AQECAHh7ea2tyZ6phZgT4B9BDKwguhlFtRC6hgt+7HbmeFsrsgAAALcwgbQGCSqGSIb3DQEHBqCBpjCBowIBADCBnQYJKoZIhvcNAQcBMB4GCWCGSAFlAwQBLjARBAwLEPDqsqAvuJdR8ckCARCAcKo/xhIswLW0MazFXGYVn+qrAFMOTEOzUiMsLNAzHJ7n4Fi1AJztcQSHIOgizHZuDXM0ULUebUaES8mmOwsH8pxi30UmX5/rtTakE5/dbVKZN7yvzt7FsEMlhfhBbloTDK0HaQcdsnL2BcnwxztXRUQ=
-ELASTIC_RESOURCES_INDEX_NAME=resources-2024-04-15
+
+ENCRYPTED_ELASTICSEARCH_URI=AQECAHh7ea2tyZ6phZgT4B9BDKwguhlFtRC6hgt+7HbmeFsrsgAAAJYwgZMGCSqGSIb3DQEHBqCBhTCBggIBADB9BgkqhkiG9w0BBwEwHgYJYIZIAWUDBAEuMBEEDMIkDoQ9C/cCDCAq1wIBEIBQ+L3OgUGeOW9rs1CWkhpBjwM4LbbVRFIWedqew4UXIeSNMJ8cO9SNe4YGCUIoKwCDYt7W7ip3VtDRRRMVvz6QJw+Eg8ugTMVs2pbNFGNvaAQ=
+ENCRYPTED_ELASTICSEARCH_API_KEY=AQECAHh7ea2tyZ6phZgT4B9BDKwguhlFtRC6hgt+7HbmeFsrsgAAAJ4wgZsGCSqGSIb3DQEHBqCBjTCBigIBADCBhAYJKoZIhvcNAQcBMB4GCWCGSAFlAwQBLjARBAxFghFNijuLqfqLD9gCARCAVyNd/BIq2x2op/mEoOL7/yNqUX51nM/fmZJ4BswSkofXSeOZFVfdE5Jf0vJLXGYZX164DdfWz7VHpYLUye+CtLztL2HBeLzcYUJNqYQzH31l0go3Cs6Drw==
+ELASTIC_RESOURCES_INDEX_NAME=resources-2024-10-17
+ES_VERSION=8
```

Comment:
Above 3 lines not related to the identify-ids-by-sierra query script, but fix stale qa-bulk config (i.e. to match qa config)